rdflib/plugins/sparql/processor.py

```diff
@@ -23,15 +23,6 @@ def prepareQuery(queryString, initNs={}, base=None) -> Query:
     return ret
 
 
-def prepareUpdate(updateString, initNs={}, base=None):
-    """
-    Parse and translate a SPARQL Update
-    """
-    ret = translateUpdate(parseUpdate(updateString), base, initNs)
-    ret._original_args = (updateString, initNs, base)
-    return ret
-
-
```

Comment:
Why are you removing this?

Reply:
Okay, I will update the changes suggested . I will add the function back again and also update the formatting of the code